### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by khannaabhi**
changes for service and function to url

### CODE REVIEW

File: modules/eventing/handle_staged.go [hunk 1]
```diff
@@ -82,8 +82,9 @@ func (m *Module) processStagedEvent(eventDoc *model.EventDocument) {
 			log.Println("Eventing: Couldn't trigger functions -", err)
 			return
 		}
-
-		result, err := m.functions.CallWithContext(ctxLocal, eventDoc.Service, eventDoc.Function, token, eventDoc)
+		var result interface{}
+		err = m.syncMan.MakeHTTPRequest(ctxLocal, utils.RequestKindDirect, "POST", eventDoc.Url, token, eventDoc, &result)
+		//result, err := m.functions.CallWithContext(ctxLocal, eventDoc.Service, eventDoc.Function, token, eventDoc)
```
Comment (sharadregoti) [inline modules/eventing/handle_staged.go]: remove this line

File: modules/realtime/helpers.go [hunk 2]
```diff
@@ -44,10 +44,11 @@ func generateEventRules(crudConfig config.Crud) []config.EventingRule {
 					// Add a new event for each db event type
 					for _, eventType := range dbEvents {
 						rule := config.EventingRule{
-							Type:     eventType,
-							Service:  serviceName,
-							Function: funcName,
-							Options:  map[string]string{"db": dbType, "col": col},
+							Type: eventType,
+							//Service:  serviceName,
```
Comment (sharadregoti) [inline modules/realtime/helpers.go]: remove commented code

File: modules/realtime/helpers.go [hunk 3]
```diff
@@ -44,10 +44,11 @@ func generateEventRules(crudConfig config.Crud) []config.EventingRule {
...
+							//Function: funcName,
+							Url:     "",
```
Comment (YourTechBud) [inline modules/realtime/helpers.go]:
```suggestion
							Url:     fmt.Sprintf("http://localhost:4122/v1/api/%s/realtime/handle", m.project),
```

File: modules/eventing/handle_staged.go [hunk 4]
```diff
@@ -82,8 +82,9 @@ func (m *Module) processStagedEvent(eventDoc *model.EventDocument) {
 			log.Println("Eventing: Couldn't trigger functions -", err)
 			return
 		}
-
-		result, err := m.functions.CallWithContext(ctxLocal, eventDoc.Service, eventDoc.Function, token, eventDoc)
+		var result interface{}
+		err = m.syncMan.MakeHTTPRequest(ctxLocal, utils.RequestKindDirect, "POST", eventDoc.Url, token, eventDoc, &result)
```
Comment (sharadregoti) [inline modules/eventing/handle_staged.go]: Remove the second parameter utils.RequestKindDirect from MakeHTTPRequest function

File: main.go [hunk 5]
```diff
@@ -81,6 +81,11 @@ var essentialFlags = []cli.Flag{
 		Usage:  "Enable consul integration",
 		EnvVar: "ENABLE_CONSUL",
 	},
+	cli.BoolFlag{
+		Name:   "enable-consul-connect",
+		Usage:  "",
```
Comment (YourTechBud) [inline main.go]:
```suggestion
		Usage:  "Enable consul connect integration",
```

File: utils/syncman/syncman.go [hunk 6]
```diff
@@ -30,14 +30,15 @@ type Manager struct {
...
+	consulService          *connect.Service
+	services               []*api.ServiceEntry
 }
 
 // New creates a new instance of the sync manager
-func New(nodeID, clusterID string, isConsulEnabled bool) (*Manager, error) {
+func New(nodeID, clusterID string, isConsulEnabled, isConsulConnectEnabled bool) (*Manager, error) {
 
 	// Create a new manager instance
 	m := &Manager{nodeID: nodeID, clusterID: clusterID}
```
Comment (YourTechBud) [inline utils/syncman/syncman.go]:
```suggestion
	m := &Manager{nodeID: nodeID, clusterID: clusterID, isConsulConnectEnabled: isConsulConnectEnabled}
```

**Pull request merged**